build.sbt

```diff
@@ -152,29 +152,6 @@ val testResourceSettings = {
   })
 }
 
-lazy val jsonConfig210 = crossProject(JVMPlatform)
-  .crossType(CrossType.Pure)
-  .in(file("config"))
-  .disablePlugins(ScriptedPlugin)
-  .disablePlugins(ScalafixPlugin)
-  .settings(publishJsonModuleSettings)
-  .settings(
-    name := "bloop-config",
-    scalaVersion := Scala210Version,
-    libraryDependencies +=
-      compilerPlugin("org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.full),
-    testResourceSettings
-  )
-  .jvmSettings(
-    testSettings,
-    target := (file("config") / "target" / "json-config-2.10" / "jvm").getAbsoluteFile,
-    libraryDependencies ++= Seq(
-      Dependencies.circeParser,
-      Dependencies.circeCore,
-      Dependencies.circeGeneric
```

Comment:
Also it looks like these three Deps are only used here. I think you can remove them from `Dependencies.scala`.

Reply:
Done! I will go through some more deps later on.